docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -330,6 +327,44 @@ On touch devices, TestCafe emulates touch events instead of mouse events.
 
 Mouse event | Touch event
 ----------- | -------------
-`mousemove` (when hovering or dragging) | `touchmove` (dragging only)
+`mousemove` (when hovering or dragging) | `touchmove` (when dragging)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Interaction Requirements
+
+TestCafe actions can interact with elements if they satisfy the following conditions:
+
+* an element is within the page's [`body`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body) or in an [`<iframe>`](#work-with-iframes). The element can be outside of a browser window as long as TestCafe can scroll to reach it.
+
+* an element is visible - its properties are as follows:
+
+    Property | Value
+    -------- | --------
+    `display`  | *not* set to `none`
+    `visibility` | set to `visible` (the default value)
+    `width`    | auto calculated as or set to > `0`
+    `height`   | auto calculated as or set to > `0`
+
+* a targeted part of the element has to be visible.  
+
+    TestCafe actions target a center of an element, or a point specified by the `offsetX` and `offsetY` options of an action. If another element obstructs the target point, the action is executed with the overlapping element (for instance, the [t.click](../../reference/test-api/testcontroller/click.md) action clicks an overlapping block).
+
+### Example: Scroll an Element into View
+
+TestCafe scrolls to reach items that are on the page but not on-screen. Therefore, TestCafe API doesn't have a designated scroll action.
```

Comment:
I don't like the `designated` word.
Can we change it with `special`, `separate`, etc.?

Reply:
edited

> dedicated